### PR TITLE
Get binary architectures use lipo -archs

### PR DESCRIPTION
**Pull request by dacaiguoguo**
Get binary architectures use `lipo -archs`

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Get binary architectures use lipo -archs  
  [dacaiguoguo](https://github.com/dacaiguoguo)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by dacaiguoguo** [comment]
Update https://github.com/CocoaPods/cocoapods-integration-specs/pull/256

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Get binary architectures use `lipo -archs`.
```
Comment (dnkoutso) [inline CHANGELOG.md]: can you please add a link to the pull request?

Comment (dacaiguoguo) [reply CHANGELOG.md]: ok

**Comment by amorde** [comment]
Do we know what version of macOS that argument was added? I found a few outdated man pages online that don't include it, so I want to make sure we aren't breaking slightly older macOS installations

**Comment by dnkoutso** [comment]
Good question and that is why maybe it was implemented as such to begin with?

**Comment by dacaiguoguo** [comment]
I do not know what version of macOS that argument was added
according to https://developer.apple.com/library/archive/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html#//apple_ref/doc/uid/TP40001051-CH1-SW2

```
Xcode 9 requires a Mac running macOS 10.13.2 or later.
Xcode 8.3.3 requires a Mac running macOS 10.12 or later.
```

Who can have an earlier version of the system, can you confirm it?

**Comment by dacaiguoguo** [comment]
https://forums.developer.apple.com/thread/125553
That argument was added at macOS  10.14.  
I think I should close this pull request.

**Comment by amorde** [comment]
Ah, that's a bummer. Thank you for looking into this. We should re-visit this again after a few macOS releases 👍

**Pull request closed without merging**